packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -800,6 +805,13 @@ class ThemeData with Diagnosticable {
       'This feature was deprecated after v2.13.0-0.0.pre.'
     )
     required this.androidOverscrollIndicator,
+    @Deprecated(
+      'No longer used by the framework, please remove any reference to it. '
+      'For more information, consult the migration guide at '
+      'https://flutter.dev/docs/release/breaking-changes/toggleable-active-color#migration-guide. '
+      'This feature was deprecated after v2.12.0-4.2.pre.',
+    )
+    required this.toggleableActiveColor,
```

Comment:
There is work being done to add a lint that deprecated + required is not allowed. 
Most of these were just updated in https://github.com/flutter/flutter/pull/102107 to use a getter method to avoid a breaking change.

Reply:
Thanks! I just made same changes for this properties